docs/installation/server-config/security.rst

```diff
@@ -29,7 +29,7 @@ You can use KSQL with a Kafka cluster in |ccloud|. For more information, see :re
 Configuring KSQL for Secured Confluent Schema Registry
 ------------------------------------------------------
 
-The following configuration is connecting KSQL with the Confluent Schema Registry over HTTPS.
+The following configuration will connect KSQL with the Confluent Schema Registry over HTTPS.
```

Comment:
suggest: `connects` -

Reply:
Do you mean:

> will connects KSQL

That doesn't sound right to me.

Reply:
sorry, meant `...The following configuration connects KSQL with...`

Reply:
Doh! Done. (How did I not get that! ;))